### PR TITLE
Gh action

### PR DESCRIPTION
**Pull request by ameshkov**
Added a GH action that will automatically create a release.

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Build and deploy
+
+on:
+  push:
+    branches:
+      - "*"
```
Comment (ngorskikh) [inline .github/workflows/workflow.yml:6]: Doesn't this mean that it will be triggered on any pushed branch?

Comment (ameshkov) [reply .github/workflows/workflow.yml:6]: Yes

Comment (ngorskikh) [reply .github/workflows/workflow.yml:6]: But shouldn't it be triggered only by pushing a tag?

Comment (ngorskikh) [reply .github/workflows/workflow.yml:6]: I see, build all branches, release only tags.

**Pull request merged**